tests/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""Unit test module."""
+import os
+import sys
+import unittest
+
+tests_dir = os.path.dirname(__file__)
+addon_dir = os.path.dirname(tests_dir)
+
+sys.path.insert(0, tests_dir)
+sys.path.insert(0, addon_dir)
+
+import tools  # noqa: E402 isort:skip
+
+from validation import (  # noqa: E402 isort:skip
+    uv_check,
+    texture_size_check,
+    texture_square_check,
+)
+
+if __name__ == '__main__':
+    # load addon module and initialize test suite
+    tools.LoadModule(os.path.join(addon_dir, '__init__.py'))
+    loader = unittest.TestLoader()
+    suite = unittest.TestSuite()
+
+    # add tests to the test suite
+    suite.addTests(loader.loadTestsFromModule(uv_check))
+    suite.addTests(loader.loadTestsFromModule(texture_size_check))
+    suite.addTests(loader.loadTestsFromModule(texture_square_check))
```

Comment:
Todos os testes vão precisar ser adicionados assim? Não teria como pegar eles do diretório?

Reply:
Realmente é meio chato ter que ficar adicionando
Em circunstâncias normais, daria pra rodar num [modo discovery](https://stackoverflow.com/questions/3295386/python-unittest-and-discovery) indicando como os arquivos de teste devem ser encontrados
Mas acho que não tem como fazer isso com a flag `-P` do blender
Vou  ver se tem como setar um diretório por código mesmo